kofta/src/app/modules/room-chat/RoomChatList.tsx

```diff
@@ -33,6 +33,7 @@ export const RoomChatList: React.FC<ChatListProps> = ({}) => {
 	// Only scroll into view if not manually scrolled to top
 	useEffect(() => {
 		isRoomChatScrolledToTop || bottomRef.current?.scrollIntoView();
+		window.addEventListener("focus", () => setNewUnreadMessages(false));
```

Comment:
you need to put this in a different useEffect or you will create hundreds of listeners, you should also stop listening on unmount